pom.xml

```diff
@@ -80,13 +81,12 @@
 
     <module>ratis-test</module>
 
-    <module>ratis-assembly</module>
     <module>ratis-examples</module>
     <module>ratis-replicated-map</module>
     <module>ratis-logservice</module>
     <module>ratis-metrics</module>
     <module>ratis-tools</module>
-    <module>ratis-docs</module>
+    <module>ratis-assembly</module>
```

Comment:
I think we should also explicitly add `ratis-docs` to dependencies of `ratis-assembly`, similarly to other Ratis artifacts:

```diff
--- pom.xml
+++ pom.xml
@@ -279,6 +279,11 @@
         <scope>test</scope>
       </dependency>

+      <dependency>
+        <artifactId>ratis-docs</artifactId>
+        <groupId>org.apache.ratis</groupId>
+        <version>${project.version}</version>
+      </dependency>
       <dependency>
         <artifactId>ratis-examples</artifactId>
         <groupId>org.apache.ratis</groupId>
diff --git ratis-assembly/pom.xml ratis-assembly/pom.xml
index 50fb8077..c5b9024a 100644
--- ratis-assembly/pom.xml
+++ ratis-assembly/pom.xml
@@ -287,6 +287,10 @@
       <groupId>org.apache.ratis</groupId>
       <artifactId>ratis-tools</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ratis</groupId>
+      <artifactId>ratis-docs</artifactId>
+    </dependency>
   </dependencies>

   <profiles>
```